framework/src/problems/FEProblemBase.C

```diff
@@ -4382,6 +4383,28 @@ FEProblemBase::backupMultiApps(ExecFlagType type)
   }
 }
 
+void
+FEProblemBase::reinitMultiApps(ExecFlagType type, std::string type_name)
+{
+  const auto & multi_apps = _multi_apps[type].getActiveObjects();
+
+  if (multi_apps.size())
```

Comment:
```suggestion
  if (hasMultiApps(type))
    const auto & multi_apps = _multi_apps[type].getActiveObjects();
```

Reply:
syntax is bad in my edit though

Reply:
Nice, I didn't know it already had a function for this!